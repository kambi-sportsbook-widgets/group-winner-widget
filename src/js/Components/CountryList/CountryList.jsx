import React from 'react';
import PropTypes from 'prop-types';
import styles from './CountryList.scss';

const CountryList = ({ children }) => (
   <ul className={styles.general}>
      {children}
   </ul>
);

CountryList.propTypes = {

   /**
    * Table rows
    */
   children: PropTypes.node,

};

export default CountryList;
