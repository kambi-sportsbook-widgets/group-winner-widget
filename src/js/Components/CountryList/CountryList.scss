:local(.general) {
   width: 100%;
   list-style: none;
}
