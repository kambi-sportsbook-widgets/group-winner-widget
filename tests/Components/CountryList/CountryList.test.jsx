/* eslint-env jest */
import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import CountryList from '../../../src/js/Components/CountryList/CountryList';

let renderer;

describe('CountryList.render()', () => {

   beforeEach(() => {
      renderer = new ReactShallowRenderer();
   });

   it('renders correctly when empty', () => {
      expect(renderer.render(
         <CountryList />
      )).toMatchSnapshot();
   });

   it('renders correctly when not empty', () => {
      expect(renderer.render(
         <CountryList>
            <li />
            <li />
         </CountryList>
      )).toMatchSnapshot();
   });

});
