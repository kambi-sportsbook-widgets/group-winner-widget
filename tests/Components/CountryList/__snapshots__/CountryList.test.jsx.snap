// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CountryList.render() renders correctly when empty 1`] = `
<ul
  className="general"
/>
`;

exports[`CountryList.render() renders correctly when not empty 1`] = `
<ul
  className="general"
>
  <li />
  <li />
</ul>
`;
